ner_v1/detectors/textual/text/text_detection.py

```diff
@@ -55,6 +59,10 @@ def __init__(self, entity_name=None):
         self.entity_name = entity_name
         self.tag = '__' + self.entity_name + '__'
 
+        # assigning values to superclass attributes
+        self.supported_languages = [ENGLISH_LANG, HINDI_LANG]
+        self.source_language_script = language_script
```

Comment:
This does not override the parent class' static member of the same name and instead assigns it as an instance attribute to this class